ompi/mpi/fortran/use-mpi-f08/Makefile.am

```diff
@@ -44,6 +45,11 @@ module_sentinel_file = \
         mod/libforce_usempif08_internal_modules_to_be_built.la \
         bindings/libforce_usempif08_internal_bindings_to_be_built.la
 
+if OMPI_FORTRAN_HAVE_C_ISO_FORTRAN
+module_sentinel_file += \
```

Comment:
I know I'm being picky here, but it confused me for a while when reading this: the `ibforce_usempif08_internal_cdesc_to_be_built.la` library is *not* a sentinel that is used to force the building of a fortran module: it is the actual library itself.  Adding it to `module_sentinel_file` and naming it `libforce_...blahblahbla` is confusing.

Also, don't you need to slurp of `libforce_ibforce_usempif08_internal_cdesc_to_be_built.la` into `libompi_usmpif08.la` (i.e., as an LT convenience library)?  Otherwise, the `[ompi_]cdesc_create_datatype` symbol won't be available to the greater OMPI code base.